### ISSUE TITLE
Tamanho do campo pss no QR Code

### ISSUE THREAD

**Issue by pcs980** [bacen/pix-api#456]
No item 1.5 da versão 2.6.0 do Manual de Padrões para Iniciação do Pix temos o novo campo `pss` dentro do ID 26:

![image](https://user-images.githubusercontent.com/12775110/136232775-d23613c7-fa98-4aca-ba8a-6dc9aa469f9d.png)

Entendo que precisamos informar os 8 dígitos do ISPB, contudo temos códigos com zeros à esquerda ou.

A dúvida é a seguinte? A definição do tamanho poderia ser alterada para 01..08 para economizar espaço?
Considerando especialmente que o ID 26 tem um tamanho limitado de 99 caracteres.

**Comment by rubenskuhl** [comment]
Eu não apoio, já vi muitos problemas de confusão de não reconhecimento de CNPJ por causa de diferenças de representação de 0s à esquerda. 

Acho que o principal a ser sacrificado aí é o infoAdicional. O que sobrar de espaço nos 99 caracteres para o infoAdicional, ok...

**Comment by LeoBOrtigoza** [comment]
concordo com o Rubens, geraria muita complexidade de desenvolvimento também

**Comment by LeoBOrtigoza** [comment]
No caso do banco do brasil que são oito 0 (00000000), como ficaria? Existem muitos casos, acho que é complicado. Para mim o ISPB é um campo string e não numérico. Quando digo complexo, é porque é interop, as regras precisam ficar muito clara para muito mais de mil PSPs, senão começamos a ter problemas em produção durante o lançamento do produto e possíveis manutenções futuras que não controlamos.

**Comment by thiagolvlsantos** [comment]
Boa tarde @pcs980,

O campo tem 8 caracteres obrigatórios exatamente para evitar qualquer interpretação diversa.

Atenciosamente,
Thiago Santos
